안드로이드 컴파일 오류 있습니다.

### Version of kakao-login libraries

<!-- 사용하고 있는 카카오 로그인 라이브러리 버전을 적어주세요 -->
2.6.0

### Version of react-native

<!-- 리액트네이티브 버전을 적어주세요 -->
0.63.1

### Platforms you faced the error (IOS or Android or both?)

<!-- 문제가 발생한 플랫폼을 적어주세요 -->
android

### Expected behavior

<!-- 정상동작 해야 하는 작업을 적어주세요 -->
정상적인 빌드

### Actual behavior

<!-- 겪고 있는 문제 상황을 적어주세요 -->
컴파일 오류

node_modules/@react-native-seoul/kakao-login/android/src/main/java/com/dooboolab/kakaologins/RNKakaoLoginsModule.java:357: error: cannot find symbol
    private void updateScopes(final ReadableArray scopes, final Promise promise) {
                                    ^
  symbol:   class ReadableArray
  location: class RNKakaoLoginsModule
/Users/jhkim/albam-connect/node_modules/@react-native-seoul/kakao-login/android/src/main/java/com/dooboolab/kakaologins/RNKakaoLoginsModule.java:366: error: cannot find symbol
            public void onAccessTokenReceived(AccessToken accessToken) {
                                              ^
  symbol: class AccessToken
2 errors

FAILURE: Build failed with an exception.

### Tested environment (Emulator? Real Device?)
안드로이드 전부 다 입니다. 

<!-- 오류를 겪고 있는 개발환경(에뮬레이터 또는 실제 기기)을 적어주세요 -->

--- a/android/src/main/java/com/dooboolab/kakaologins/RNKakaoLoginsModule.java
+++ b/android/src/main/java/com/dooboolab/kakaologins/RNKakaoLoginsModule.java
@@ -27,6 +27,7 @@ import com.facebook.react.bridge.LifecycleEventListener;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.bridge.Promise;
 
@@ -35,6 +36,7 @@ import com.kakao.auth.ISessionCallback;
 import com.kakao.auth.AccessTokenCallback;
 import com.kakao.auth.KakaoSDK;
 import com.kakao.auth.Session;
+import com.kakao.auth.authorization.accesstoken.AccessToken;
 import com.kakao.network.ErrorResult;
 import com.kakao.usermgmt.UserManagement;
 import com.kakao.usermgmt.callback.LogoutResponseCallback;

